src/game/common/audio/audioeventinfo.h

```diff
@@ -23,11 +23,19 @@
 
 enum AudioType : int32_t
```

Comment:
`enum AudioEventType`

Reply:
not quite, there's a unclarity on if its EVENT actually, for now leaving as is